irohad/torii/command_service.cpp

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 #include "torii/command_service.hpp"
```

Comment:
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
We should include it with `"` symbol